src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -600,18 +606,27 @@ public ResponseEntity<Void> moveModification(@PathVariable("studyUuid") UUID stu
                                                         @PathVariable("modificationUuid") UUID modificationUuid,
                                                         @Nullable @Parameter(description = "move before, if no value move to end") @RequestParam(value = "beforeUuid") UUID beforeUuid) {
         studyService.assertCanModifyNode(studyUuid, nodeUuid);
-        studyService.reorderModification(studyUuid, nodeUuid, modificationUuid, beforeUuid);
+        studyService.moveModification(studyUuid, nodeUuid, List.of(modificationUuid), beforeUuid);
         return ResponseEntity.ok().build();
     }
 
     @PutMapping(value = "/studies/{studyUuid}/nodes/{nodeUuid}", produces = MediaType.APPLICATION_JSON_VALUE)
-    @Operation(summary = "For a list of network modifications passed in body, duplicate and append them to current node")
+    @Operation(summary = "For a list of network modifications passed in body, duplicate or cut, then append them to target node")
     @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The modification list has been updated. Modifications in failure are returned.")})
-    public ResponseEntity<String> duplicateModifications(@PathVariable("studyUuid") UUID studyUuid,
+    public ResponseEntity<String> updateModifications(@PathVariable("studyUuid") UUID studyUuid,
```

Comment:
This name will be used in the new endpoints I'm working on (createModification / updateModification / deleteModifications), can you choose some more explicit, like copyOrCut or something like that ?

Reply:
fixed